docs/developer-manual/terms-of-service.md

```diff
@@ -0,0 +1,157 @@
+---
+id: terms of service
+sidebar_position: 5
+slug: /tos
+---
+
+# Terms of Service
+
+<!-- markdownlint-disable MD036 -->
+<!-- To be linked to from `"tos":` -->
+
+*MM DD YYYY*
+
+<!-- markdownlint-restore -->
+
+These terms of service, together with any documents and additional terms they
+incorporate by reference (collectively, these “Terms”), are entered into between
+Orcfax Ltd. (the “Oracle,” “we,” “us,” and “our”) and you or the company or
+other legal entity that you represent (“you” or “your”).
+
+Please read these Terms carefully as they govern your use of our site located at
+Orcfax.io and all associated sites (the “Website”) and our Services
+(defined below) and describe your rights and obligations and our disclaimers and
+limitations of legal liability. By accessing or using any part of the Website or
+the Services, you agree to become bound by the terms and conditions of these
+Terms.
+
+If you use the Services on behalf of a company, organization (including a
+decentralized autonomous organization or “DAO”) or other entity then “you”
+includes you and that entity, and you represent and warrant that (a) you are an
+authorized representative of the entity with the authority to bind the entity to
+these Terms, and (b) you agree to these Terms on the entity’s behalf. If you do
+not agree to these Terms or do not have authority to bind your organization on
+whose behalf you are using the Services to these Terms, you must not access or
+use our Website or the Services.
+
+We reserve the right, in our sole discretion, to modify these Terms from time to
+time. If we make changes, we will provide you with notice of such changes using
+commercially reasonable means, such as by providing a notice through the Website
+or our Services or updating the date at the top of these Terms. Unless we say
+otherwise in our notice, any modifications are effective immediately, and your
+continued use of the Website or our Services will confirm your acceptance of the
+changes. If you do not agree to the amended Terms, you must stop using our
+Services.
+
+## SERVICES
+
+The Website enables users to access open source documentation, content, and
+services, including (without limitation) resources, data and computation
+services provided by a network of nodes directly to smart contracts integrating
+Orcfax feeds, and information and resources for the community comprising the
+Orcfax Network (collectively such services, information, APIs, and resources
+related to the Orcfax network are referred to as the “Services”).
+
+## YOUR REPRESENTATIONS AND WARRANTIES; CONDITIONS
+
+You represent and warrant that you: (i) have the necessary technical expertise
+and ability to review and evaluate the security, integrity and operation of any
+Fact Tokens that you decide to acquire, sell or use; (ii) have the knowledge,
```

Comment:
> use of LINK _might_ be synonymous with requesting/buying data

You red this correctly! Because we have long discussed a utility of FACT being its use for purchasing feed publication... I figured that we would also need to use this kind of language. With that said, I support changing the language to reflect the reality at present.